css/stylesheet.css

```diff
@@ -0,0 +1,803 @@
+* {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
```

Comment:
Vendor prefixing is not needed. http://caniuse.com/#search=border-box

Always research solutions if you are unsure. It takes 5 seconds to look at can I use.